core/src/main/java/org/apache/syncope/core/quartz/AbstractTaskJob.java

```diff
@@ -176,4 +183,15 @@ public void execute(final JobExecutionContext context) throws JobExecutionExcept
     protected boolean hasToBeRegistered(final TaskExec execution) {
         return false;
     }
+
+    @Override
+    public void interrupt() throws UnableToInterruptJobException {        
+        Thread thread = this.runningThread.getAndSet(null);
+        if (thread != null) {
+            LOG.info("Interrupting job time {} ", new Date().toString());
```

Comment:
Please consistently use `LOG.debug` and format the date according to `SyncopeConstants` format.